yellowbrick/features/pca.py

```diff
@@ -172,26 +181,57 @@ def transform(self, X, y=None, **kwargs):
 
     def draw(self, **kwargs):
         X = self.pca_features_
-
         if self.proj_dim == 2:
             self.ax.scatter(X[:, 0], X[:, 1], c=self.color, cmap=self.colormap)
+            if self.proj_features:
+                x_vector = self.pca_components_[0]
+                y_vector = self.pca_components_[1]
+                max_x = max(X[:, 0])
+                max_y = max(X[:, 1])
+                for i in range(self.pca_components_.shape[1]):
+                    plt.arrow(
+                        x=0, y=0,
+                        dx=x_vector[i] * max_x,
+                        dy=y_vector[i] * max_y,
+                        color='r', head_width=0.05,
+                        width=0.005,
+                    )
+                    plt.text(
+                        x_vector[i] * max_x * 1.05,
+                        y_vector[i] * max_y * 1.05,
+                        self.features_[i], color='r'
+                    )
         if self.proj_dim == 3:
             self.fig = plt.figure()
-            self.fig = self.fig.add_subplot(111, projection='3d')
-
-            self.ax = self.fig.scatter(X[:, 0], X[:, 1], X[:, 2], c=self.color,
-                                       cmap=self.colormap)
+            self.ax = self.fig.add_subplot(111, projection='3d')
+            self.ax.scatter(X[:, 0], X[:, 1], X[:, 2],
+                            c=self.color, cmap=self.colormap)
+            if self.proj_features:
+                x_vector = self.pca_components_[0]
+                y_vector = self.pca_components_[1]
+                z_vector = self.pca_components_[2]
+                max_x = max(X[:, 0])
+                max_y = max(X[:, 1])
+                max_z = max(X[:, 1])
+                for i in range(self.pca_components_.shape[1]):
+                    self.ax.plot(
+                        [0, x_vector[i] * max_x],
+                        [0, y_vector[i] * max_y],
+                        [0, z_vector[i] * max_z],
+                        color='r'
+                    )
+                    self.ax.text(
+                        x_vector[i] * max_x * 1.05,
+                        y_vector[i] * max_y * 1.05,
+                        z_vector[i] * max_z * 1.05,
+                        self.features_[i], color='r'
```

Comment:
You were saying something before about having difficulty with the 3D vectors; has that been resolved? 

Also, what is the 1.05 multiplier here? Just offsetting the text by that much from the arrow?

Reply:
I found a way to plot 3D arrows in matplolib [here](https://stackoverflow.com/questions/22867620/putting-arrowheads-on-vectors-in-matplotlibs-3d-plot)  but it requires to create a class that inherits from FancyArrowPatch, so I decided to keep it simple and just plot a line.

And yes, is to get the offset

Reply:
Awesome, thanks for clarifying!